gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -276,6 +312,14 @@ def configure_interfaces(self, context, resource_data):
             LOG.error(msg)
             raise Exception(msg)
 
+        try:
+            status = self._log_forwarding(mgmt_ip)
+        except Exception:
+            msg = ("Configuration of log forwarding has failed. Reason: %r" %
+                   status.json())
+            LOG.error(msg)
+            return status.json()
```

Comment:
Can we convey the same bug to visibility team, else they might expect this to be present on chain launch. We can file a bug for this.
If this is a serious bug, then why we are proceeding further, why not return error.
Are these changes only for fixing chain launch, will there be another PR for visibility.